dotnet/samples/HomeAutomation/AppConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.Configuration;
+
+namespace HomeAutomation;
+
+/// <summary>
+/// Loads configuration upon instantiation.
+/// </summary>
+internal sealed class AppConfiguration
```

Comment:
Can we rename this to something more specific like AzureAiOptions? If we need more config later we can create models for the sections we need.

Reply:
Also remove the constructor and make this a simple model that is bound to with the options patern.

Reply:
Done